packages/orchestrator-ui-components/src/rtk/endpoints/streamMessages.ts

```diff
@@ -56,9 +67,10 @@ const streamMessagesApi = orchestratorApi.injectEndpoints({
 
                 const state = getState() as RootState;
                 const { orchestratorWebsocketUrl } = state.orchestratorConfig;
+                const validCacheTags = Object.values(CacheTags);
 
                 // Starts the websocket
-                const webSocket = getWebSocket(orchestratorWebsocketUrl);
+                const webSocket = await getWebSocket(orchestratorWebsocketUrl);
```

Comment:
Add `ORCHESTRATOR_WEBSOCKET_URL` to the env.example as well

Reply:
nevermind, I didn't see it

Reply:
I think t's already there